backend/kernelCI_app/utils.py

```diff
@@ -152,3 +152,19 @@ def validate_comparison_op(self, op):
     def get_comparison_op(self, filter, op_type="orm"):
         idx = self.comparison_op_type_idx[op_type]
         return self.comparison_ops[filter["comparison_op"]][idx]
+
+    def get_grouped_filters(self):
+        grouped_filters = {}
+
+        for f in self.filters:
+            field = f["field"]
+            value = f['value']
+            if field not in grouped_filters:
+                grouped_filters[field] = f
+                grouped_filters[field]['value'] = value
```

Comment:
f already has the `value` property right?

Reply:
yes, `f` is something like this `{'field': 'valid', 'value': 'false', 'comparison_op': 'exact'}`

Reply:
Now I got it, line 164 is unnecessary